### PR TITLE
feat: USDV page

### PR DESCRIPTION
**Pull request by chefjackson**
---

## PR-Codex overview
This PR adds support for the USDV widget, updates dependencies, and introduces new components and hooks.

### Detailed summary
- Added `@usdv/usdv-widget` dependency
- Updated `useUsdvMintAvailable` hook
- Introduced `ClientOnly` and `PoweredBy` components
- Modified `CurrencyLogo` component
- Added `ExternalCurrencyLink` component
- Updated translations for USDV-related messages

> The following files were skipped due to too many changes: `apps/web/src/components/SearchModal/CurrencySearch.tsx`, `apps/web/src/pages/usdv.tsx`

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

File: apps/web/package.json [hunk 1]
```diff
@@ -77,6 +77,7 @@
     "@sentry/nextjs": "^7.52.1",
     "@snapshot-labs/snapshot.js": "^0.4.40",
     "@tanstack/react-query": "^5.28.4",
+    "@usdv/usdv-widget": "^0.9.8",
```
Comment (ChefJerry) [inline apps/web/package.json:80]: this package is not open-sourced

Comment (chefjackson) [reply apps/web/package.json:80]: sry what do you mean? You mean the code is minified?

Comment (chefjackson) [reply apps/web/package.json:80]: Disclaimer added

**Comment by chefjackson** [comment]
@SocketSecurity ignore npm/@tinyhttp/app@1.3.0
@SocketSecurity ignore npm/@tinyhttp/etag@1.3.0
@SocketSecurity ignore npm/@tinyhttp/send@1.3.0

**Review by ChefMomota — APPROVED** [review]

**Pull request merged**